src/test/components/TrackMemory.test.js

```diff
@@ -117,18 +121,13 @@ describe('TrackMemory', function() {
     expect(ctx.__flushDrawLog()).toMatchSnapshot();
   });
 
-  it('has the correct selectors into useful parts of the component for the samples profile', function() {
-    const { canvas } = setup();
-    expect(canvas.exists()).toBe(true);
-  });
```

Comment:
I removed this test because of `ensureExists` is used in `setup`, so the case should be handled already.